app/controllers/accountants_controller.rb

```diff
@@ -5,30 +5,15 @@ class AccountantsController < ApplicationController
   before_action :find_patient, only: [:edit]
 
   def index
-    @patients = paginate_results(pledged_patients)
-    @entry_name = t('common.patient').downcase
-  end
-
-  def search
-    have_search = params[:search].present?
-    have_clinic = params[:clinic_id].present?
-
-    if have_search || have_clinic
-      partial = pledged_patients
-
-      partial = partial.where(clinic_id: params[:clinic_id]) if have_clinic
-      partial = partial.search(params[:search], search_limit: nil) if have_search
-
-      results = partial
-    else
-      results = pledged_patients
-    end
-
-    # when we search, counter says 'results' instead of 'patients'
-    @entry_name = t('accountants.results')
-    @results = paginate_results results
-
-    respond_to { |format| format.js }
+    @patients = Patient.where(pledge_sent: true)
+                       .includes(:clinic)
+                       .includes(:fulfillment)
+                       .order(pledge_sent_at: :desc)
+                       .page(params[:page] || 1)
+
+    @patients = @patients.where(clinic_id: params[:clinic_id]) if params[:clinic_id].present?
+    @patients = @patients.search(params[:search]) if params[:search].present?
+    @patients
```

Comment:
ha this is much cleaner! and I guess this really is the way you're supposed to do it, huh?

Reply:
🤷  no idea honestly! I spent a lot of time flipping thru kaminari docs and I'm still not totally sure I got it right.